### PR TITLE
[finishes #187748153] hero-and-categories-sections-ui-implementation

### PR DESCRIPTION
**Pull request by amin-leon**
## Purpose

Describe the purpose of the PR

## Changes Made

Describe the changes made

## Testing Instructions

List all testing instruction, if availbale

## Related Issues

Lists all related issues

## Checklist

Please review the following checklist and make sure all tasks are complete before submitting:

- [ ] Code follows the project's coding standards
- [ ] Changes are covered by tests
- [ ] Documentation is updated (if applicable)
- [ ] All tests pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "lucide-react": "^0.387.0",
```
Comment (Mag-codes) [inline package.json]: can we use one Library for Icons here we have lucide-react and react-icons. I think react-icons might be good

**Review by hozayves — CHANGES_REQUESTED** [review]
Hi NP,
I appreciate the amazing effort you've put in, but I have some feedback:

- Could you align your work more closely with the design in Figma?
- I'm curious how your design would accommodate 100 categories.

**Comment by amin-leon** [comment]
> Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> 
> * Could you align your work more closely with the design in Figma?
> * I'm curious how your design would accommodate 100 categories.

How about on figma for 100 ? can u show me how it can be handled?

**Comment by amin-leon** [comment]
> Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> 
> * Could you align your work more closely with the design in Figma?
> * I'm curious how your design would accommodate 100 categories.

can you show the difference? those categories cards, right ?? i have explained it why i changed them

**Comment by hozayves** [comment]
> > Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> > 
> > * Could you align your work more closely with the design in Figma?
> > * I'm curious how your design would accommodate 100 categories.
> 
> can you show the difference? those categories cards, right ?? i have explained it why i changed them

I wasn't on the call, so I'm not sure why you decided to change the design.

**Comment by amin-leon** [comment]
> > > Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> > > 
> > > * Could you align your work more closely with the design in Figma?
> > > * I'm curious how your design would accommodate 100 categories.
> > 
> > 
> > can you show the difference? those categories cards, right ?? i have explained it why i changed them
> 
> I wasn't on the call, so I'm not sure why you decided to change the design.

you were not! okay

**Comment by hozayves** [comment]
> > > Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> > > 
> > > * Could you align your work more closely with the design in Figma?
> > > * I'm curious how your design would accommodate 100 categories.
> > 
> > 
> > can you show the difference? those categories cards, right ?? i have explained it why i changed them
> 
> I wasn't on the call, so I'm not sure why you decided to change the design.

You need to write it in a way that anyone, at any time, can read and understand it.

**Comment by hozayves** [comment]
> > > > Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> > > > 
> > > > * Could you align your work more closely with the design in Figma?
> > > > * I'm curious how your design would accommodate 100 categories.
> > > 
> > > 
> > > can you show the difference? those categories cards, right ?? i have explained it why i changed them
> > 
> > 
> > I wasn't on the call, so I'm not sure why you decided to change the design.
> 
> you were not! okay

NP, during the demo time, I wasn't in the meeting. I joined a bit late.

**Comment by amin-leon** [comment]
> > > > Hi NP, I appreciate the amazing effort you've put in, but I have some feedback:
> > > > 
> > > > * Could you align your work more closely with the design in Figma?
> > > > * I'm curious how your design would accommodate 100 categories.
> > > 
> > > 
> > > can you show the difference? those categories cards, right ?? i have explained it why i changed them
> > 
> > 
> > I wasn't on the call, so I'm not sure why you decided to change the design.
> 
> You need to write it in a way that anyone, at any time, can read and understand it.

what to write and understand? mn be specific boy!

**Review by hozayves — APPROVED** [review]

**Review by Mag-codes — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

File: src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ function Navbar() {
     }, []);
 
     return (
-        <div className='flex flex-col bg-blackColor md:bg-whiteColor md:text-blackColor text-whiteColor font-roboto w-full 2xl:items-center sticky top-0'>
+        <div className='flex flex-col bg-blackColor md:bg-whiteColor md:text-blackColor text-whiteColor font-roboto w-full 2xl:items-center fixed z-50'>
```
Comment (Favor-star) [inline src/components/navbar/Navbar.tsx:40]: This was a bug we removed yesterday on call. Yet you're returning it. why?

File: src/containers/Arrivals/NewArrivals.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  const [currentPage, setCurrentPage] = useState(0);
+
+  console.log("helllooooo", data)
+
+  useEffect(() => {
+    console.log("Fetching data: isLoading?", isLoading, "Error:", error, "Products:", data);
+  }, [isLoading, error, data]);
+
+  if (isLoading) {
+    console.log("Loading...");
```
Comment (patrickhag) [inline src/containers/Arrivals/NewArrivals.tsx:21]: You should those log in codebase?

File: src/containers/Arrivals/NewArrivals.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  if (error) {
+    console.log("Error loading products:", error);
+    // Optionally display an error indicator
+    return null;
+  }
+
+  const productsData: ProductResponse = data as unknown as ProductResponse;
+
+  if (!productsData) {
+    console.log("No products data received");
```
Comment (patrickhag) [inline src/containers/Arrivals/NewArrivals.tsx:34]: even here

File: src/containers/FeaturedProducts/FeaturedProducts.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  if (error) {
+    console.log("Error loading products:", error);
+    // Optionally display an error indicator
+    return null;
+  }
+
+  const productsData: ProductResponse = data as unknown as ProductResponse;
+
+  if (!productsData) {
+    console.log("No products data received");
```
Comment (Favor-star) [inline src/containers/FeaturedProducts/FeaturedProducts.tsx:31]: Console.log? Is this okay? How a standard user gonna see this?

**Review by patrickhag — CHANGES_REQUESTED** [review]
consider removing those console logs man

**Review by Favor-star — CHANGES_REQUESTED** [review]
ESPECIALLY REMOVE THAT POSITION FIXED TOU SET ON NAVBAR, BRING IT BACK TO STICKY.
You was not around yesterday that is why; bbut may ask me reasons later.

**Review by patrickhag — APPROVED** [review]

**Pull request merged**